docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,279 @@
 
-TODO: write this.
+## Overview
+
+Registering a WEC with an ITS follows the same process as registering a managed cluster with an OCM hub. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+The instructions below provide a comprehensive registration process focused on KubeStellar-specific considerations. For additional OCM registration details, refer to the [official Open Cluster Management documentation](https://open-cluster-management.io/docs/getting-started/installation/register-a-cluster/).
+
+### Terminology Mapping
+
+| OCM Term             | KubeStellar Equivalent                        |
+|----------------------|-----------------------------------------------|
+| **OCM Hub**          | **ITS** (Inventory and Transport Space) |
+| **OCM Managed Cluster** | **WEC** (Workload Execution Cluster) |
+| **Klusterlet**       | **Klusterlet** |
+
+## Prerequisites
+
+Before registering a WEC, ensure you have:
+
+1. **An existing ITS** with OCM cluster manager running
+2. **A running Kubernetes cluster** that will serve as the WEC
+3. **Network connectivity** from the WEC to the ITS (HTTPS connections must be possible)
+4. **`kubectl` access** to both the WEC and ITS
+5. **`clusteradm` CLI tool** installed on the machine where you will run the registration commands ([installation guide](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/))
+6. **Sufficient permissions** to create resources in both clusters (typically cluster-admin or equivalent)
+
+For a complete understanding of how WEC registration fits into the overall KubeStellar architecture, see [The Full Story](user-guide-intro.md#the-full-story).
+
+To verify your ITS is ready for WEC registration:
+
+```shell
+# Check if the ITS is accessible (this returns a short result)
+kubectl --context <its-context> get ServiceAccount default
+
+# Verify OCM cluster manager is running (look for absence of Pending or failing Pods)
+kubectl --context <its-context> get pods -n open-cluster-management-hub
+
+# Check for the customization-properties namespace
+kubectl --context <its-context> get ns customization-properties
+```
+
+## Registration Process
+
+### Step 1: Get the Registration Token
+
+Obtain a registration token from the ITS:
+
+```shell
+# Get the registration token from the ITS
+clusteradm --context <its-context> get token
+```
+
+This command outputs a `clusteradm join` command that you will use in the next step. The token and apiserver URL from this command will be used in the join command.
+
+### Step 2: Join the WEC to the ITS
+
+**Important:** Before executing the join command, determine if you need additional flags based on your environment. The basic command below may not work for all cluster types.
+
+Execute the join command on the WEC to initiate the registration process. The exact command depends on your environment:
```

Comment:
s/:/./

The colon implies that what follows is an elaboration on the remark before the colon, but that is not what actually follows.